docs/ref/models/expressions.txt

```diff
@@ -252,6 +252,16 @@ The ``Func`` API is as follows:
         A class attribute that denotes the character used to join the list of
         ``expressions`` together. Defaults to ``', '``.
 
+    .. attribute:: arity
+
+        .. versionadded:: 1.10
+
+        A class attribute that denotes the exact number of expressions
+        expected by the function. If this attribute is set and the function
+        is called with the different number of expressions, ``TypeError``
+        will be raised.
+        Defaults to ``None``.
```

Comment:
If this line is meant as a separate paragraph, it needs a blank line above it, otherwise, please flow it up to the previous line. Please include in release notes too.